perfkitbenchmarker/providers/gcp/gcp_dpb_dataflow.py

```diff
@@ -200,6 +199,13 @@ def GetMetadata(self):
     basic_data['dpb_job_id'] = self.job_id
     return basic_data
 
+  def GetJobStatus(self):
```

Comment:
Move this method to the child class. It doesn't belong here conceptually.

Reply:
It was added here because retrieving job status applies to any dataflow provider (parent or child). It's the same reason I added job_id in this parent class. Depending on the benchmark, one might need to get job status to perform other operations like cleanup.

Reply:
Sorry I got confused, since we don't launch the jobs using the gcloud CLI I thought you couldn't interact with them in this way, but you are right.